db/structure.sql

```diff
@@ -49,8 +49,7 @@ SET search_path = public, pg_catalog;
 
 CREATE TYPE format_enum AS ENUM (
     'html',
-    'markdown',
-    'text'
```

Comment:
No that's just an accident - that branch is using a different database and apparently it had that value missing for some reason. I've fixed it (and the random field order changes in user) now and also rebased over current master.